lib/AST/Decl.cpp

```diff
@@ -1031,7 +1031,7 @@ unsigned PatternBindingDecl::getPatternEntryIndexForVarDecl(const VarDecl *VD) c
     ++Result;
   }
   
-  assert(0 && "PatternBindingDecl doesn't bind the specified VarDecl!");
+  llvm_unreachable("PatternBindingDecl doesn't bind the specified VarDecl!");
```

Comment:
Ok. I was reaching for it since it explicitly advertises itself as a replacement for assert(0) and the latter causes an unreachable code warning. What should I use instead?